fixtures/events/performance_problems/consecutive-http/consecutive-http-basic.json

```diff
@@ -1126,7 +1119,7 @@
     ]
   },
   "sdk": {
-    "name": "sentry.javascript.react",
+    "name": "sentry.javascript.node",
```

Comment:
I didn't really want to create a new event with a non-frontend implementation, so I just replaced the sdk name to something that isn't considered frontend and dropped the frontend related measurements

I can create a real event if someone feels strongly about it.